domain/domain.go

```diff
@@ -392,8 +392,10 @@ func (m *MockFailure) getValue() bool {
 	return m.val
 }
 
-type etcdBackend interface {
+// EtcdBackend is used for judging a storage is a real TiKV.
```

Comment:
Whether the storage can run GC has nothing to with whether it uses etcd. So, I think it's better to use 2 interfaces, base on the 'minimum interface principle'.